show-magnifier/src/main/java/com/esri/arcgismaps/sample/showmagnifier/screens/MainScreen.kt

```diff
@@ -17,34 +17,37 @@
 package com.esri.arcgismaps.sample.showmagnifier.screens
 
 import android.app.Application
-import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Scaffold
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
+import com.arcgismaps.mapping.ArcGISMap
+import com.arcgismaps.mapping.BasemapStyle
+import com.arcgismaps.mapping.Viewpoint
+import com.arcgismaps.mapping.view.MapViewInteractionOptions
+import com.arcgismaps.toolkit.geocompose.MapView
+import com.arcgismaps.toolkit.geocompose.MapViewpointOperation
 import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
-import com.esri.arcgismaps.sample.showmagnifier.components.ComposeMapView
-import com.esri.arcgismaps.sample.showmagnifier.components.MapViewModel
 
 /**
  * Main screen layout for the sample app
  */
 @Composable
-fun MainScreen(sampleName: String, application: Application) {
-    // create a ViewModel to handle MapView interactions
-    var mapViewModel = MapViewModel(application)
+fun MainScreen(sampleName: String) {
+    // Create an ArcGISMap and Viewpoint
+    val arcGISMap = ArcGISMap(BasemapStyle.ArcGISTopographic)
+    val viewpoint = Viewpoint(34.056295, -117.195800, 1000000.0)
```

Comment:
The set operation for viewpoint might look unclear with `Set(viewpoint = viewpoint)`, maybe rename the variable or  create the viewpoint object directly at the set operation. 

```suggestion
    val californiaViewpoint = Viewpoint(34.056295, -117.195800, 1000000.0)
```